pkg/pillar/scripts/device-steps.sh

```diff
@@ -15,7 +15,7 @@ ZTMPDIR=/var/tmp/zededa
 DPCDIR=$ZTMPDIR/DevicePortConfig
 FIRSTBOOTFILE=$ZTMPDIR/first-boot
 GCDIR=$PERSISTDIR/config/ConfigItemValueMap
-AGENTS0="logmanager ledmanager nim nodeagent domainmgr"
+AGENTS0="logmanager newlogmgr ledmanager nim nodeagent domainmgr"
```

Comment:
Does it mean we will be running logmanager and newlogmanager side by side for some time? When do you plan a cut over?